File: samples_generated_code/samples_3/phi4/baseline_code_c/code_row_66.c
#include <stdio.h>
#include <stdlib.h>

int getValueFromArray(int arr[], int size, int index) {
    if (index >= 0 && index < size) {
        return arr[index];
    }
    // Return -1 to indicate an error for out-of-bounds access.
    return -1;
}

int main() {
    int size = 5;  // Example array size
    int arr[size] = {10, 20, 30, 40, 50};  // Example array

    printf("Enter the index (0-%d): ", size - 1);
    int index;
    if (scanf("%d", &index) != 1) {
        return EXIT_FAILURE;
    }

    int result = getValueFromArray(arr, size, index);

    // Using result as exit code
    return result;
}

File: samples_generated_code/samples_3/athene/baseline_code_java/code_row_66.java
import java.util.Scanner;

public class Main {
    public static int getValueFromArray(int[] arr, int size, int index) {
        if (index < 0 || index >= size) {
            return -1;
        }
        return arr[index];
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] array = {1, 2, 3, 4, 5};
        int size = array.length;
        System.out.println("Enter an index:");
        int index = scanner.nextInt();
        int value = getValueFromArray(array, size, index);
        System.exit(value);
    }
}